kubernetes/kubernetes.go

```diff
@@ -91,7 +91,25 @@ const direct = "direct"
 const unreachable = "unreachable"
 
 //nolint llll
-const kbURL string = "https://support.cloudability.com/hc/en-us/articles/360008368193-Kubernetes-Metrics-Agent-Error-Messages"
+const kbErrorURL string = "https://help.apptio.com/en-us/cloudability/product/k8s-metrics-agent.htm"
+const kbProvisionURL string = "https://help.apptio.com/en-us/cloudability/product/k8s-cluster-provisioning.htm"
+const apiEndpoint string = "https://metrics-collector.cloudability.com"
+
+const forbiddenError string = "Error retrieving upload URI: 403"
+const uploadURIError string = "Error retrieving upload URI"
+const transportError string = "Network transport issues are potentially blocking the agent from " +
+	"contacting the metrics collection API.\n" +
+	"Please confirm that the metrics-agent is able to establish a connection to: %s"
+const apiKeyError string = "Current Cloudability API Key needs to be re-enabled before re-provisioning " +
```

Comment:
Would change "Current Cloudability API Key needs to be re-enabled before reprovisioning the metrics-agent as detailed here: %s/n" to something like "Current Cloudability API Key is expired.  For more details see %s.\n". It's a slight correction since we aren't reactivating the same api key, but generating a new one

Reply:
I think the provisioning page doesn't have info about expired API keys, so we need some info about re-activating the expired key, because part of these messages is that, customers might not actually go to the help pages, so we want to give as much info in as short of a blurb as possible.

Reply:
That's a good point. I think we should make sure that we're not saying we are reactivating the same api key, but instead generating a new one. We've gotten customer questions in the past about the particularities of provisioning so I just want to make sure our messaging is consistent about what is actually happening.

Reply:
How does this sound?

> Current Cloudability API Key is expired and access needs to be re-enabled before re-provisioning the metrics-agent as detailed here

Reply:
Love it 👍